src/bundles/comms.stanza

```diff
@@ -111,10 +111,23 @@ public pcb-bundle can-logical :
 ; SPI - Serial Peripheral Interface
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+doc: \<DOC>
+SPI Bundle Pin Names
+A list of possible names for the SPI interface. Note that in order to connect
+SPI interfaces automatically, the constituent pins need to be present on all
+objects to be connected. 
+<DOC>
+
 public pcb-enum jsl/bundles/comms/SPIPins :
   SPI-CS
   SPI-MISO
   SPI-MOSI
+  SPI-COPI
+  SPI-CIPO
```

Comment:
`COPI` and `MOSI` are synonymous.
Same with `CIPO` and `MISO`. 

I don't think we should have two bundle definitions for the same thing with different names. 

If we want to use `CIPO/COPI` instead of `MISO/MOSI` - thats fine. Lets replace them - but lets not do both.

Reply:
Keeping COPI / CIPO